keylime/keylime_agent.py

```diff
@@ -426,23 +424,34 @@ def __init__(self, server_address, RequestHandlerClass, agent_uuid, contact_ip,
         self.rsapublickey_exportable = crypto.rsa_export_pubkey(
             self.rsaprivatekey)
 
-        if os.path.isfile(certname):
-            logger.info("Using existing mTLS cert in %s", certname)
-            with open(certname, "rb") as f:
-                mtls_cert = x509.load_pem_x509_certificate(f.read(), backend=default_backend())
-        else:
-            logger.info("No mTLS certificate found, generating a new one")
-            agent_ips = [server_address[0]]
-            if contact_ip is not None:
-                agent_ips.append(contact_ip)
-            with open(certname, "wb") as f:
-                # By default generate a TLS certificate valid for 5 years
-                valid_util = datetime.datetime.utcnow() + datetime.timedelta(days=(360 * 5))
-                mtls_cert = crypto.generate_selfsigned_cert(agent_uuid, rsa_key, valid_util, agent_ips)
-                f.write(mtls_cert.public_bytes(serialization.Encoding.PEM))
-
-        self.mtls_cert_path = certname
-        self.mtls_cert = mtls_cert
+        self.mtls_cert_enabled = config.getboolean('cloud_agent', 'mtls_cert_enabled', fallback=False)
+        if self.mtls_cert_enabled :
```

Comment:
It is not common to have a space after `:` in Python can you fix that in your PR?
```suggestion
        if self.mtls_cert_enabled:
```